nuxeo-connect-client/src/main/java/org/nuxeo/connect/connector/http/proxy/NashornProxyPacResolver.java

```diff
@@ -97,13 +101,15 @@ protected Reader getRemotePacBodyReader() throws IOException {
         }
 
         if (fileCache.getValue() == null) {
-            HttpClient client = new HttpClient();
-            GetMethod method = new GetMethod(ConnectUrlConfig.getProxyPacUrl());
-            int status = client.executeMethod(method);
-            if (status == HttpStatus.SC_OK) {
-                fileCache.saveValue(method.getResponseBodyAsString());
-            } else {
-                throw new IOException("Unable to get pac file");
+            try (CloseableHttpClient httpClient = HttpClients.createDefault()) {
+                try (CloseableHttpResponse httpResponse = httpClient.execute(
+                        new HttpGet(ConnectUrlConfig.getProxyPacUrl()))) {
+                    if (httpResponse.getStatusLine().getStatusCode() == HttpStatus.SC_OK) {
+                        fileCache.saveValue(EntityUtils.toString(httpResponse.getEntity()));
+                    } else {
+                        throw new IOException("Unable to get pac file");
```

Comment:
`+ httpResponse.getStatusLine()` in the exception message for better diagnostics.